runtime/lua/vim/treesitter.lua

```diff
@@ -542,7 +542,7 @@ function M.inspect_tree(opts)
   vim.bo[b].buflisted = false
   vim.bo[b].buftype = 'nofile'
   vim.bo[b].bufhidden = 'wipe'
-  vim.bo[b].filetype = 'query'
+  vim.bo[b].filetype = 'playground'
```

Comment:
I renamed `vim.treesitter.playground` to `vim.treesitter.dev` in https://github.com/neovim/neovim/commit/c48b1421af28d0317c807bca00c7e2fff97d9ad0 to avoid new jargon. "dev mode" or "dev tools" is already used and is more generic, so we don't need "playground".

For filetypes I think we need a `ts.` or `treesitter.` prefix. Using unqualified names like "query" and "playground" lacks context and risks conflict with other filetypes.
```suggestion
  vim.bo[b].filetype = 'treesitter.inspector'
```

Reply:
`query` _is_ the filetype, though, in the standard Vim sense. (I'm not fully convinced we need a separate filetype to solve the actual issue, which is avoiding attaching the query linter to the tree view.)

Reply:
Are there any other filetype names with a `.` in them? From what I can see in https://github.com/neovim/neovim/tree/master/runtime/ftplugin wouldn't it make more sense to name it something like `tsinspector` or `treesitterinspector`? Generally it seems that most filetype names like this are made by just concatenating string.

Reply:
Indeed, we should not use a `.` as that has a distinct interpretation (multiple filetypes). `tsinspector` sounds fine to me.